.github/workflows/manage-github-issue-for-outdated-pods.yml

```diff
@@ -0,0 +1,34 @@
+name: Check outdate pod and create issue
+
+on: 
+  schedule:
+    - cron: '0 0 * * 1'
+  workflow_dispatch:
+
+jobs:
+  check-outdated-pods:
+    runs-on: macos-latest
+    name: Check outdate pods and create issue if it doesn't exist
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check outdated pods and create issue
+        id: check-outdated-pods-and-create-issue
+        uses: rudderlabs/github-action-updated-pods-notifier@main
+        with:
+          outdated-pod-names: "Amplitude"
+          directory: "Example"
+          title: "fix: update Amplitude SDK to the latest version"
+          assignee: "desusai7"
+          labels: "outdatedPod"
+          color: "FBCA04"
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
github.token why is this used instead of the default one? secrets.GITHUB_TOKEN

Reply:
I've updated it to `secrets.GITHUB_TOKEN`.

Somewhere in the doc, it was mentioned to use the `github.token` hence I used that.